pontoon/base/utils.py

```diff
@@ -332,6 +333,7 @@ def require_AJAX(f):
     """
     AJAX request required decorator
     """
+    @functools.wraps(f)
```

Comment:
I doubt it. Well, we'll see in New Relic if any of the important transactions is missing.